src/calculateRentalCost.js

```diff
@@ -3,8 +3,24 @@
  *
  * @return {number}
  */
+const DAY_RENT = 40;
+const LONG_TERM = 7;
```

Comment:
```suggestion
const longTerm = 7;
```
use CamelCase for const